shell/platform/darwin/ios/framework/Source/FlutterObservatoryPublisher.mm

```diff
@@ -113,6 +113,11 @@ - (void)publishServiceProtocolPort:(NSString*)uri {
 
   if (err != 0) {
     FML_LOG(ERROR) << "Failed to register observatory port with mDNS.";
+    if (@available(iOS 14.0, *)) {
+      FML_LOG(ERROR) << "Make sure the 'NSBonjourServices' key is set in your Info.plist for '"
```

Comment:
Can we point out it's only needed for Debug and Profile configuration `Info.plist`s?  At some point we should write up some info on the website, but in the meantime...

Reply:
Sg, done.

Reply:
Debug and Profile, not just Debug.